en/12_Project_Governance/05_Release_policy.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Release policy
+summary: Outline the release lifecycle and support commitment for Silverstripe CMS
+icon: code-branch
+---
+
+# Silverstripe CMS release policy
+
+This document outlines the release policy for Silverstripe CMS. It applies to all [commercially supported modules](/project_governance/supported_modules/).
+
+Community modules are not covered by this policy. Modules in the silverstripe GitHub organisation that are not commercially supported are updated on a best effort basis.
+
+> [!NOTE]
+> Refer to our [definition of public API](/project_governance/public_api/).
+
+The purpose of this policy is to provide clear guidance on the types of releases, their cadence, scope, and support lifecycle, ensuring predictability and stability for our users and developers.
```

Comment:
I think these should be the other way around but I'm not gonna hold up merging for it.

Reply:
Updated